RPIPs/RPIP-33.md

```diff
@@ -198,6 +197,8 @@ for the duration of the proposal process. In order to be eligible to propose, no
 for the purposes of rewards, voting and collateral requirements. Locked RPL SHALL NOT be counted towards thresholds for
 withdrawing RPL.
 
+By default, locking RPL for any purpose will be disabled. Node operator's will opt-in to performing governance activities by enabling the locking of RPL from their node or primary withdrawal address. On the condition that [RPIP-31: RPL Withdrawal Address](./RPIP-31.md) is ratified and implemented, if the RPL withdrawal address is set then, you will only be able to enable or disable RPL locking using this address.
```

Comment:
`locking RPL for any purpose` is too strong I think. Maybe "for any purpose defined in this RPIP" or enumerate them? Just don't want to accidentally have this interact with future proposals.

Reply:
yeah good point. At this stage, the functionality is quite broad but from an RPIP perspective we should only have it in relation to this RPIP and include any future behaviour in future RPIPs.

I will update.